nautobot/utilities/tests/test_filters.py

```diff
@@ -660,35 +662,36 @@ def test_multi_value_time_filter(self):
         self.assertEqual(self.filters["timefield__gte"].exclude, False)
 
     def test_multiple_choice_filter(self):
-        self.assertIsInstance(self.filters["multiplechoicefield"], django_filters.MultipleChoiceFilter)
-        self.assertEqual(self.filters["multiplechoicefield"].lookup_expr, "exact")
-        self.assertEqual(self.filters["multiplechoicefield"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__n"].lookup_expr, "exact")
-        self.assertEqual(self.filters["multiplechoicefield__n"].exclude, True)
-        self.assertEqual(self.filters["multiplechoicefield__ie"].lookup_expr, "iexact")
-        self.assertEqual(self.filters["multiplechoicefield__ie"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__nie"].lookup_expr, "iexact")
-        self.assertEqual(self.filters["multiplechoicefield__nie"].exclude, True)
-        self.assertEqual(self.filters["multiplechoicefield__ic"].lookup_expr, "icontains")
-        self.assertEqual(self.filters["multiplechoicefield__ic"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__nic"].lookup_expr, "icontains")
-        self.assertEqual(self.filters["multiplechoicefield__nic"].exclude, True)
-        self.assertEqual(self.filters["multiplechoicefield__isw"].lookup_expr, "istartswith")
-        self.assertEqual(self.filters["multiplechoicefield__isw"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__nisw"].lookup_expr, "istartswith")
-        self.assertEqual(self.filters["multiplechoicefield__nisw"].exclude, True)
-        self.assertEqual(self.filters["multiplechoicefield__iew"].lookup_expr, "iendswith")
-        self.assertEqual(self.filters["multiplechoicefield__iew"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__niew"].lookup_expr, "iendswith")
-        self.assertEqual(self.filters["multiplechoicefield__niew"].exclude, True)
-        self.assertEqual(self.filters["multiplechoicefield__re"].lookup_expr, "regex")
-        self.assertEqual(self.filters["multiplechoicefield__re"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__nre"].lookup_expr, "regex")
-        self.assertEqual(self.filters["multiplechoicefield__nre"].exclude, True)
-        self.assertEqual(self.filters["multiplechoicefield__ire"].lookup_expr, "iregex")
-        self.assertEqual(self.filters["multiplechoicefield__ire"].exclude, False)
-        self.assertEqual(self.filters["multiplechoicefield__nire"].lookup_expr, "iregex")
-        self.assertEqual(self.filters["multiplechoicefield__nire"].exclude, True)
+        for field in ["multiplechoicefield", "charchoicefield"]:
```

Comment:
Am I missing something, or is there no test case for the existing `"choicefield"` filter?

Reply:
Nope, or not that I could find

Reply:
Would you mind adding one then? :-)